src/service/subFolderElementHandler.js

```diff
@@ -0,0 +1,33 @@
+'use strict'
+const StandardHandler = require('./standardHandler')
+const mc = require('../utils/metadataConstants')
+const path = require('path')
+
+class SubFolderElementHandler extends StandardHandler {
+  handleAddition() {
+    super.handleAddition()
+    if (!this.config.generateDelta) return
+
+    const fileName = this.splittedLine
+      .slice(this.splittedLine.length - 1)
+      .join(path.sep)
+
```

Comment:
Should we had the copying of the file itself (and not only the meta file) ? 
```
this._copyFiles(
  path.join(this.config.repo, fileName),
  path.join(this.config.output, fileName)
 )
```

Reply:
No. It's no necessary. 
I'm testing it and it works fine for me only with _getParsedPath method adding the last .replace: 

class SubFolderElementHandler extends StandardHandler {
  _getParsedPath() {
    return path.parse(
      this.splittedLine
        .slice(this.splittedLine.length - 1)
        .join(path.sep)
        .replace(mc.META_REGEX, '')
        **.replace(this.suffixRegex, '')**
    )
  }
}

I've update the PR removing useful override and test review.